src/pjpersist/serializers.py

```diff
@@ -57,7 +57,7 @@ def write(self, obj):
 class DateTimeSerializer(serialize.ObjectSerializer):
 
     # XXX: timezone?
-    fmt = "%Y-%m-%dT%H:%M:%S"
```

Comment:
BTW, what's up with datetimes stored without %f?

Reply:
I wonder why class.fmt does not use the FMT* constants

Reply:
You are right, this does not fail. As for the second question I have not seen code from `serializerz.py` used anywhere in the code so maybe these ones are _custom_ and can be different from the main code in `serialize.py`